src/common/booking/SelectAnotherBookingLink.js

```diff
@@ -39,7 +40,7 @@ const SelectAnotherBookingLink = (props: Props) => (
                 external={false}
                 type="primary"
               >
-                Select another booking
+                <Trans t={__('smartfaq.select_another_booking')} />
```

Comment:
I see, that there is no more translated text in this file, but maybe for consistency I would use something like `smartfaq.select_another_booking.link`. What do you think?

Reply:
I don't know. Does it make a difference? :D